src/Form/LegacyConsumer/Actions/UploadFilesAction.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Give\Form\LegacyConsumer\Actions;
+
+
+use Give\Form\LegacyConsumer\Traits\HasFilesArray;
+use Give\Framework\FieldsAPI\File;
+use function wp_check_filetype;
+use function wp_generate_attachment_metadata;
+use function wp_handle_upload;
+use function wp_insert_attachment;
+use function wp_update_attachment_metadata;
```

Comment:
@ravinderk As a note, you don't need to import root functions. This is really only necessary if you're importing namespaced functions.